### DISCUSSION TITLE
Location for Identity config

### DISCUSSION THREAD

**Discussion by reubenhEROAD** [featurehub-io/featurehub#949] [Q&A] [answered]
I am self hosting the 'Streaming Scalable Deployment' of Featurehub in k8s, using the provided helm chart. I want to implement identity federation with Azure OAuth2. I am following the instructions in the docs, but I can't figure out where this config is meant to be defined. Can someone please direct me where to put this config? Thanks. 

https://docs.featurehub.io/featurehub/latest/identity.html

**Comment by rvowles** [comment, marked as answer]
Heya @reubenhEROAD - there is a section in the helm chart called `environmentVars` for each application. The docs indicate this is configuration for the  Management Repository,  so around line 143  of the  values.yaml you will see the  database config? If you  add your  Azure config in a similar way  there, it will copy the  appropriate  config as  env vars. There is a mechanism to allow you to refer  to a secretRef as well but i'm not sure if  thats what you are after?

**Reply by reubenhEROAD** [reply]
Hey @rvowles that seems obvious now, but I wasn't sure. Thank you for that!
